tests/integration/test_wu.py

```diff
@@ -170,11 +166,8 @@ def setUp(self):
         self.name = "Test Nutzer"
         self.mail = "foo@bar.baz"
 
-        self.user = self.create_user_ldap_patched(
-            uid=self.nutzer.unix_account,
-            name=self.name,
-            mail=self.mail,
-        )
+        self.user = self.create_user_ldap_patched(uid=self.nutzer.unix_account,
```

Comment:
Please go through these changes manually instead of pressing „automatic refactoring“ in your IDE. If the diff can be `-1` instead of `+2-4`, it should be preferred.